idea_town/frontend/static-src/app/views/experiment-page.js

```diff
@@ -22,7 +22,7 @@ export default PageView.extend({
 
   events: {
     'click [data-hook=install]': 'install',
-    'click [data-hook=uninstall]': 'uninstall',
+    'click [data-hook=uninstall]': 'renderUninstallSurvey',
```

Comment:
Since you added `id` attributes above, you may be able to convert these 3 things to something like:
```js
'click #uninstall-button': 'renderUninstallSurvey'
```